gem/lib/earlgrey/configure_earlgrey.rb

```diff
@@ -353,7 +354,8 @@ def copy_swift_files
       dst_header = File.join(dst_root, src_header_name)
 
       src_swift_name = 'EarlGrey.swift'
-      src_swift = File.join(src_root, src_swift_name)
+      src_swift = File.join(src_root, "Swift-" + "#{swift_version}/" + src_swift_name)
```

Comment:
When using interpolation (`#{}`), you don't use +. Also the path separator should never be hard coded as that's the reason we use the Join method. See the corrected code below.

```ruby
 File.join(src_root, "Swift-#{swift_version}", src_swift_name)
```